src/Acuminator/Acuminator.Analyzers/StaticAnalysis/RowChangesInEventHandlers/DiagnosticWalker.cs

```diff
@@ -26,13 +26,16 @@ private class DiagnosticWalker : CSharpSyntaxWalker
 			private SymbolAnalysisContext _context;
 			private readonly SemanticModel _semanticModel;
 			private readonly PXContext _pxContext;
+			private readonly bool _reversed;
```

Comment:
@dropsonic _reversed is not very clear name in code. Imho more meaningful name can be used. Even better option is a enum. Something like RowChangesAnalysisMode

Reply:
@SENya1990 I agree, will try to come up with another name